src/assets/scss/layout/layout.scss

```diff
@@ -31,9 +31,11 @@
   }
 }
 
-.footer-styling {
-  padding-bottom: 3em;
-  padding-top: 5em;
+.page-wrapper {
+  position: absolute;
+  width: 100%;
+  padding-top: var(--pf-c-page__header--MinHeight);
```

Comment:
My only concern here is that `var (--pf-c-page__header--MinHeight)` may not give the correct values we are looking for, as we previously had issues with using PF variables and minimum padding rules

The previous padding top was to ensure that the content is not hidden under the header, so would padding by the page header's minimum height guarantee that in all situations?

Reply:
Thanks for pointing that out! You're right, using the min-height doesn't guarantee that the body won't be blocked by the header. We don't actually have a need for the header to grow larger than the min-height, so I'll just specify the exact height for the `.header` class:

```
.header {
  position: fixed;
  top: 0em;
  left: 0em;
  right: 0em;
  width: 100%;
  height: var(--pf-c-page__header--MinHeight); // New
...
}
```